portfolio/src/main/webapp/index.html

```diff
@@ -2,32 +2,83 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
     <title>Ihsan Olawale's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="assets/css/main.css">
     <script src="script.js"></script>
     <script src="quiz.js"></script>
+    <noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
   </head>
-  <body>
-    <div id="content">
-      <h1>Ihsan Olawale's Portfolio</h1>
-      <p>This is Ihsan Olawale's portfolio.</p>
-      <p>I, Ihsan Olawale, am a STEP intern at Google. By the end of the internship I will have created a web project. This is practice.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-      <p>Here's what I look like, at least when looking at my Google security photo:</p>
-      <img src="/images/me.png" alt="What I look like" width="300" height="300">
-      <p>After seeing what I look like, you might be interested to know more about me. Don't worry, I considered this during the design phase, and presented an opportunity to cycle through assorted facts.</p>
-      <button onclick="addRandomFact()">Learn about me</button>
-      <div id="fact-container"></div>
-      <div id="quiz-container">
-	  <p>
-	      If you clicked the button I presented previously, you would have recognised you were trolled. I'm sorry for that, and have presented a set of multiple choice questions. Guess your way through to learn more about me:
-	  </p>
-	  <div id="questions-container"></div>
-	  <button type="button" onclick="checkSubmission()">Check Your Answer</button>
-	  <div id="submit-container"></div>
+  <body class="is-preload">
+    <header id="header">
+      <h1>Ihsan Olawale</h1>
+      <nav>
+	<ul>
+	  <li><a href="#intro">Intro</a></li>
+	  <li><a href="#looks">What I Look Like</a></li>
+	  <li><a href="#more">Learn More</a></li>
+	</ul>
+      </nav>
+    </header>
+    <section id="intro" class="main style1 dark fullscreen">
+      <div class="content">
+	<header>
```

Comment:
indentation is a bit off but dw about it for now :)

Reply:
Oh ok this is the only indentation mistake that can be attributed to me, the others are auto-added.

Reply:
Ah vim autoreplaced 8 spaces with a tab I think, which might explain the inconsistent indentation here. I'll look into this later.